persistence-cassandra/core/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraProvider.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import akka.actor.ActorSystem
+
+/**
+ * Internal API
+ */
+private[lagom] class CassandraProvider(system: ActorSystem) {
```

Comment:
This seems to be used only for read-side related config loading. I think `CassandraProvider` is a very generic name that could lead to confusion.
There's a `ReadSideConfig` in `persistence-core` and a `CassandraReadSideSessionProvider` in  `persistence-cassandra-core`.